lib/salus/config.rb

```diff
@@ -55,6 +57,11 @@ def initialize(configuration_files = [], ignore_ids = [])
       # Check if any of the values are actually pointing to envars.
       final_config = fetch_envars(final_config)
 
+      Salus::PluginManager.load_plugins
```

Comment:
We should probably move this to lib/salus.rb.  That would ensure that the plugins are loaded and ready as we start enabling other hooks (beyond the config hook).